src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -353,13 +354,18 @@ def create(
             self,
             nqn: str,
             rbd_image_name: str,
+            rbd_image_size: str,
```

Comment:
this change needs to regenerate the openapi spec file to update the documentation. you can do that by going to the dashboard folder and running `tox -e openapi-fix`

Reply:
@nizamial09 something strange about it
I ran the exact command you mentioned above, and it seems like it just deletes the whole part about nvme API entirely(you can see I even naively commited it haha).
I tried to run the same command in `main` branch and it behaves the same. also tried `tox -e openapi-check` and it only passes when the nvme section is deleted . I wonder if there might be a recent change that was merged and causing this? maybe something I should do when I run it and I don't understand? wdyt?

Reply:
yup, you are right. its because of the missing proto files in the local. I am writing a tox script here that generates the proto files locally which ensures that the nvmeof controllers are loaded properly while generating the openapi docs so that it also documents (or not remove the nvmeof controllers). Probably you can apply it to your local.
https://github.com/ceph/ceph/pull/61846/files

```
(maybe you'll have to init the nvmeof submodule in git)
tox -e gen-nvmeof-proto
tox -e openapi-fix
```